specification/definitions/ServiceRate.json

```
@@ -0,0 +1,67 @@
+{
+  "type": "object",
+  "required": [
+    "type"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "type": {
+      "$ref": "../types/ServiceRate.json"
+    },
+    "attributes": {
+      "type": "object",
+      "required": [
+        "code",
+        "weight_min",
+        "weight_max",
+        "length_max",
+        "width_max",
+        "height_max",
+        "volume_max",
+        "price",
+        "purchase_price",
+        "fuel_surcharge"
```

Let's use the same required fields as our API.
Although I think the prices can be required (that's one of the reasons why we have this endpoint).
```suggestion
        "price",
        "purchase_price"
```